zng/docs/spec.md

```diff
@@ -12,6 +12,32 @@
 > Also, we are contemplating reducing the number of primitive types, e.g.,
 > the number of variations in integer types.
 
+* [1. Summary](#1-summary)
+* [2. The ZNG data model](#2-the-zng-data-model)
+* [3. ZNG Binary Format (ZNG)](#3-zng-binary-format-zng)
+  + [3.1 Control Messages](#31-control-messages)
+    - [3.1.1 Typedefs](#311-typedefs)
+      - [3.1.1.1 Record Typedef](#3111-record-typedef)
+      - [3.1.1.2 Array Typedef](#3112-array-typedef)
+      - [3.1.1.3 Set Typedef](#3113-set-typedef)
+      - [3.1.1.4 Union Typedef](#3114-union-typedef)
+      - [3.1.1.5 Alias Typedef](#3115-alias-typedef)
+    - [3.1.2 End-of-stream Markers](#312-end-of-stream-markers)
+  + [3.2 ZNG Value Messages](#32-zng-value-messages)
+* [4. ZNG Text Format (TZNG)](#4-zng-text-format-tzng)
+  + [4.1 ZNG Control Messages](#41-zng-control-messages)
+    - [4.1.1 Type Binding](#411-type-binding)
+    - [4.1.2 Type Alias](#412-type-alias)
+    - [4.1.3 Application-specific Payload](#413-application-specific-payload)
+  + [4.2 Type Grammar](#42-type-grammar)
+  + [4.3 ZNG Values](#43-zng-values)
+    - [4.3.1 Character Escape Rules](#431-character-escape-rules)
+    - [4.3.2 Value Syntax](#432-value-syntax)
+  + [4.4 Examples](#44-examples)
+* [5. Related Links](#5-related-links)
+
+## 1. Summary
```

Comment:
s/Summary/Introduction/

Reply:
🤷‍♂️  Works for me. I actually did waffle between "Summary" and "Introduction" and didn't feel strongly either way, so I flipped the coin and went with "Summary". Since I have at least one tiebreaker vote now, I've flipped it to "Introduction" per your suggestion.